### PR TITLE
Adding link to online gateway

### PR DESCRIPTION
**Pull request by Kcchouette**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -3,12 +3,17 @@ const hashString = 'Hello from IPFS Gateway Checker'
 
 const $results = document.querySelector('#results')
 
+function returnHtmlLink (gateway) {
+	let gatewayTitle = gateway.split(hashToTest)[0]
+	return '<a title="' + gatewayTitle + '" href="' + gateway '" >' + gateway + '</a>';
```
Comment (VictorBjelkholm) [inline app.js]: It's missing a `+` here I think.

Would probably be easier to write it like this:

return `<a title="${gatewayTitle}" href="${gateway}">${gateway}</a>`

Comment (VictorBjelkholm) [reply app.js]: Updated previous comment as I made a mistake with the formatting.

File: app.js [hunk 2]
```diff
@@ -3,12 +3,17 @@ const hashString = 'Hello from IPFS Gateway Checker'
...
+}
+
 function addNode (gateway, online, title) {
 	const para = document.createElement('div')
 	let node
 	if (online) {
 		node = document.createElement('strong')
-		node.innerText = '✅ - Online  - ' + gateway
+		node.innerText = '✅ - Online  - ' + returnHtmlLink(gateway)
```
Comment (VictorBjelkholm) [inline app.js]: I don't think this will work, `innerText` only sets text. Would have to change to be `innerHTML` here.

File: app.js (hunk 1 above)
Comment (Kcchouette) [reply app.js]: indeed, thank you

File: app.js (hunk 2 above)
Comment (Kcchouette) [reply app.js]: indeed

**Comment by jamiew** [comment]
@Kcchouette would you be willing to rebase this with master and I'll merge? Nice patch, just out of date now. Thanks!

**Pull request closed without merging**